templates/base.html

```diff
@@ -1,51 +1,38 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <meta charset='utf-8'>
     <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Ethical Eating</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1'>
-    <link rel='stylesheet' type='text/css' media='screen' href="{{ url_for('static', filename='css/main.css') }}">
-    <link rel="icon" href="data:,">
-    <script></script>
+    <link rel='stylesheet' type='text/css' media='screen' href="style.css">
```

Comment:
I think this needs to be reverted to `<link rel='stylesheet' type='text/css' media='screen' href="{{ url_for('static', filename='css/main.css') }}">` in order for all our css to show up when loading.